README.cs.md

```diff
@@ -32,10 +32,11 @@ Nainstalujte závislosti:
 * pango
 * cairo
 * gdk-pixbuf2 (volitelné: oznamovací oblast)
+* [swaybg] (volitelné: tapeta)
 * [scdoc] (volitelné: manuálové stránky) \*
 * git (volitelné: informace o verzi) \*
 
-_\* Závislost pouze pro sestavení_
+_\* Závislost pouze pro kompilaci_
```

Comment:
Both "sestavení" and "kompilaci" are correct here, but the latter is probably more common.